packages/http-server/src/server.ts

```diff
@@ -126,7 +130,7 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
           }
         });
 
-        return TE.fromIOEither(() =>
+        return IOE.fromEither(
```

Comment:
In case you didn't notice, see that we have a function less now

Reply:
Interesting